static/app/components/events/eventStatisticalDetector/transactionFrequencyChart.tsx

```diff
@@ -0,0 +1,175 @@
+
+  const {transaction, breakpoint} = event?.occurrence?.evidenceData ?? {};
+
+  const eventView = EventView.fromLocation(location);
+  eventView.query = `event.type:transaction transaction:"${transaction}"`;
+  eventView.dataset = DiscoverDatasets.METRICS;
+
+  const {start: beforeDateTime, end: afterDateTime} = useRelativeDateTime({
+    anchor: breakpoint,
+    relativeDays: 14,
+  });
+
+  eventView.start = (beforeDateTime as Date).toISOString();
+  eventView.end = (afterDateTime as Date).toISOString();
+  eventView.statsPeriod = undefined;
+
+  // The evidence data keys are returned to us in camelCase, but we need to
+  // convert them to snake_case to match the NormalizedTrendsTransaction type
+  const normalizedOccurrenceEvent = Object.keys(
+    event?.occurrence?.evidenceData ?? []
+  ).reduce((acc, key) => {
+    acc[camelToUnderscore(key)] = event?.occurrence?.evidenceData?.[key];
+    return acc;
+  }, {}) as NormalizedTrendsTransaction;
+
+  const {data, isLoading} = useGenericDiscoverQuery<
+    {
+      data: EventsStatsData;
+      meta: MetaType;
+    },
+    DiscoverQueryProps
+  >({
+    route: 'events-stats',
+    location,
+    eventView,
+    orgSlug: organization.slug,
+    getRequestPayload: () => ({
+      // Manually inject y-axis for events-stats because
+      // getEventsAPIPayload doesn't pass it along
+      ...eventView.getEventsAPIPayload(location),
+      yAxis: 'count()',
+    }),
+  });
+
+  return (
+    <DataSection>
+      <TransitionChart loading={isLoading} reloading>
+        <TransparentLoadingMask visible={isLoading} />
+        <Chart
+          statsData={data?.data ?? []}
+          evidenceData={normalizedOccurrenceEvent}
+          start={eventView.start}
+          end={eventView.end}
+          chartLabel="TPM"
+        />
+      </TransitionChart>
+    </DataSection>
+  );
+}
+
+interface ChartProps {
+  chartLabel: string;
+  end: string;
+  evidenceData: NormalizedTrendsTransaction;
+  start: string;
+  statsData: EventsStatsData;
+}
+
+function Chart({statsData, evidenceData, start, end, chartLabel}: ChartProps) {
+  const theme = useTheme();
+  const router = useRouter();
+
+  const resultSeries = transformEventStats(statsData, chartLabel);
+
+  const needsLabel = true;
+  const intervalSeries = getIntervalLine(
+    theme,
+    resultSeries || [],
+    0.5,
+    needsLabel,
+    evidenceData,
+    IntervalLineFormat.REGRESSION_EPM
+  );
+
+  const series = [...resultSeries, ...intervalSeries];
+
+  const durationUnit = getDurationUnit(series);
+
+  const chartOptions: Omit<LineChartProps, 'series'> = {
+    tooltip: {
+      valueFormatter: (value, seriesName) => {
+        return tooltipFormatter(value, aggregateOutputType(seriesName));
+      },
+    },
+    yAxis: {
+      minInterval: durationUnit,
+      axisLabel: {
+        color: theme.chartLabel,
+        formatter: (value: number) =>
+          axisLabelFormatter(value, 'rate', undefined, durationUnit),
```

Comment:
I think you're querying the `count()`, rendering it as a rate per second here.

Reply:
Thanks, you're right, changed to `epm()`